lib/matplotlib/contour.py

```diff
@@ -761,15 +764,20 @@ def __init__(self, ax, *args, **kwargs):
         Draw contour lines or filled regions, depending on
         whether keyword arg *filled* is ``False`` (default) or ``True``.
 
-        The first three arguments must be:
+        Call signature::
+
+            ContourSet(ax, levels, allsegs, [allkinds], **kwargs)
```

Comment:
I would just pick some reasonable argument names (levels, allsegs and allkinds are fine, but you can also change them if you want as they were positional-only up to now) and transform this into a regular signature without custom parsing.

Reply:
I agree with the proposed change, but will leave it to a separate PR.

It's probably simple enough. I did not touch this, because there's a bit of code paths to change with `_process_args` and maybe subclasses. I try to keep the docstring updates separated from really non-trivial code changes. This eases reviewing as well as backporting.